airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -31,7 +31,6 @@
 
 from airflow.utils.log.logging_mixin import LoggingMixin
 
-
 class SalesforceHook(BaseHook, LoggingMixin):
```

Comment:
This `LoggingMixin` can be removed since you already inherit it from the BaseHook

Reply:
I revert the line removing in Salesforce_hook as this PR is not related at all to anything in Salesforce. The change was induced by a conflict when I rebased on master a few weeks ago.